src/macros.rs

```diff
@@ -256,7 +256,7 @@ macro_rules! intrinsics {
         #[cfg(all(any(windows, target_os = "uefi"), target_arch = "x86_64", not(feature = "mangled-names")))]
         mod $name {
             #[no_mangle]
-            #[cfg_attr(all(not(windows), not(target_vendor = "apple")), linkage = "weak")]
+            #[cfg_attr(not(all(windows, target_env = "gnu")), linkage = "weak")]
```

Comment:
It should likely also check `not(target_abi = "llvm"`, how do you test it? I'd like to test with gnullvm.

Reply:
Just testing this crate locally? You should be able to `cd` to `testcrate` and `cargo test --target ...`.

Reply:
Oh, I thought this issue would manifest only during Rust build, gonna look into it this weekend.

Reply:
The rust-lang/rust update of compiler-builtins merged, so it should also be available to test with the next rustc nightly (couple hours after 0:00 UTC)